Shu/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.10-slim-bullseye
+COPY requirements.txt /home
+COPY test.py /home
+RUN pip freeze > requirements.txt
+RUN python -m pip install -r requirements.txt
```

Comment:
```suggestion
```

I'm confused by why these line are here, given that the first line is printing nothing into a file located at `/requirements.txt` and then the second line is installing that empty file. Neither of these lines are installing the dependency in the existing `requirements.txt` that you copied to `/home/requirements.txt`. Even if they were, the behavior is strange as you wouldn't want to overwrite the contents of `/home/requirements.txt`.

These aren't doing anything useful, so they should be removed.